detox/e2e/test/autocomplete/at_mention.e2e.js

```diff
@@ -167,4 +167,24 @@ describe('Autocomplete', () => {
         // * Expect at mention autocomplete not to contain associated user suggestion
         await expect(element(by.id(`autocomplete.at_mention.item.${user.id}`))).not.toExist();
     });
+
+    it('should be able to select at mention multiple times', async () => {
```

Comment:
I added this test case to TM4J

```suggestion
    it('MM-T3409_11 should be able to select at mention multiple times', async () => {
```